modules/core/src/main/scala/doobie/util/query.scala

```diff
@@ -252,6 +264,8 @@ object query {
 
   }
 
+  type PreparedExecutionUpdate[A] = PreparedExecutionWithResultSet[A] => PreparedExecutionWithResultSet[A]
```

Comment:
IMO I don't feel we don't need a type alias for this? My don't use type aliases too often as I find it makes APIs harder to understand as it requires additional jump-to-definition to understand what a function is doing.